frappe/public/js/frappe/views/reports/report_view.js

```diff
@@ -778,12 +777,24 @@ frappe.views.ReportView = class ReportView extends frappe.views.ListView {
 
 		let doctype_fields = frappe.meta.get_docfields(this.doctype).filter(standard_fields_filter);
 
+		// filter out docstatus field from picker
+		let std_fields = frappe.model.std_fields.filter( df => !in_list('docstatus', df.fieldname));
+
+		// add status field derived from docstatus, if status is not a standard field
+		if (!frappe.meta.has_field(this.doctype, 'status')) {
+			doctype_fields = [{
+				label: __('Status'),
+				fieldname: 'docstatus',
+				fieldtype: 'Data'
+			}].concat(doctype_fields, std_fields);
```

Comment:
`std_fields` should be concatenated even if there is no `status` field in the doctype right?